types/dadata/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@definitelytyped/dtslint/dt.json",
+  "rules": {
+    "comment-format": false,
+    "export-just-namespace": false,
+    "interface-name": false,
+    "no-any-union": false,
+    "no-unnecessary-class": false
+  }
```

Comment:
Are these needed? New packages should leave all the default lint rules on unless there's a good reason. I don't see any classes in the code, so I guess that no-unnecessary-class, at least, is not needed at all.

Reply:
I removed the extra rules that I laid down for the future. Thank you